src/gui/src/settings/options-window.cpp

```diff
@@ -342,6 +343,151 @@ OptionsWindow::OptionsWindow(Profile *profile, ThemeLoader *themeLoader, QWidget
 	ui->comboImagePositionVideoH->setCurrentIndex(positionsH.indexOf(settings->value("imagePositionVideoH", "left").toString()));
 	ui->lineImageBackgroundColor->setText(settings->value("imageBackgroundColor", QString()).toString());
 
+	ui->checkRememberDrawer->setChecked(settings->value("Zoom/rememberDrawer", true).toBool());
+	ui->checkRememberGeometry->setChecked(settings->value("Zoom/rememberGeometry", true).toBool());
+
+	if (settings->value("firstload", true).toBool()) {
```

Comment:
Note that this won't work for:
* Existing users (when I tried for examples)
* New users that don't open the settings window from the "first load" window

I think you'd be better off introducing another flag, such as "firstload-settings" or something.

Reply:
I thought the ZoomWindow should fall back on the intended default configuration when settings are not available. Maybe that got broken. I will investigate.

Reply:
So when I run Grabber for the first time, the settings are written without opening the options-window. [Here](https://github.com/Bionus/imgbrd-grabber/files/7269855/grabber.log) is an example log of the console output. Does it behave differently on other platforms?

I will add that flag for existing users. I thought "first load" meant after installation or upgrade, not just when writing a new config file.